templates/tools/dockerfile/interoptest/grpc_interop_go/Dockerfile.template

```diff
@@ -14,7 +14,7 @@
   # See the License for the specific language governing permissions and
   # limitations under the License.
   
-  FROM golang:1.11
+  FROM golang:1.15
```

Comment:
FTR note that you'll need to update the interop matrix images when the next release happens.